### PR TITLE
Announcing KubeDB v2022.10.18

### PR DESCRIPTION
**Pull request by spectro30**
Signed-off-by: Md. Alif Biswas <alif@appscode.com>

### CODE REVIEW

**Review by rakibulhossain — DISMISSED** [review]
LGTM!

File: content/post/kubedb-v2022.10.18/index.md [hunk 1]
```diff
@@ -0,0 +1,785 @@
...
+- proxysql
+- pgbouncer
+- schema-manager
+---
+
+We are pleased to announce the release of [KubeDB v2022.10.18](https://kubedb.com/docs/v2022.10.18/setup/). This post lists all the major changes done in this release since the last release.
+This release offers some major features like **Externally Managed Secret**, **Percona XtraDB OpRequests**, **ProxySQL OpsRequests**, **Redis Sentinel OpsRequests**, **PostgreSQL Logical Replication**, **MongoDB Hidden Members Support**, **Autoscaler Support for MySQL, Postgres, Redis, Redis Sentinel & PerconaXtraDB** etc.
+It also contains various improvements and bug fixes. `Kubernetes version 1.25` is supported in KubeDB from this release.  You can find the detailed changelogs [here](https://github.com/kubedb/CHANGELOG/blob/master/releases/v2022.10.18/README.md).
+
+## Externally Managed Secret
```
Comment (ArnobKumarSaha) [inline content/post/kubedb-v2022.10.18/index.md:34]: `Externally Managed Auth Secret` ?

File: content/post/kubedb-v2022.10.18/index.md [hunk 2]
```diff
@@ -0,0 +1,785 @@
...
+- mysql
+- opensearch
+- opensearch-dashboard
+- percona
+- percona-xtradb
+- postgresql
+- redis
+- proxysql
+- pgbouncer
+- schema-manager
```
Comment (ArnobKumarSaha) [inline content/post/kubedb-v2022.10.18/index.md:27]: Add `autoscaler` tag

File: content/post/kubedb-v2022.10.18/index.md [hunk 3]
```diff
@@ -0,0 +1,785 @@
...
+We have added the most recent MySQL versions.
+Now you can create a Standalone , Group Replication, Innodb Cluster, Read Replica and Semi-sync Cluster with `MySQL 8.0.31`
+You can get started with `mysql:8.0.31` by applying this simple yaml
+### BugFix and Improvement
+Several bugs in mysql ops request including mysql version upgrading and reconfigure TLS and Custom configuration are fixed.
+OpsRequest for innodb cluster were improved. And in this release innodbCluster is more stable in provisioning and failover.
+Now kubedb managed MySQL instance can automatically recover when some group member has extra transaction on it. It will clone from the Primary instance to recover.
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
```
Comment (ArnobKumarSaha) [inline content/post/kubedb-v2022.10.18/index.md:181]: move this yaml to MySQL New version support section

File: content/post/kubedb-v2022.10.18/index.md [hunk 4]
```diff
@@ -0,0 +1,785 @@
...
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 10Gi
+  terminationPolicy: WipeOut
+```
+
+## MariaDB
+Along with externally managed secret support, We have fixed bugs and made improvements in several places of MariaDB Provisioner and OpsRequests.
```
Comment (ArnobKumarSaha) [inline content/post/kubedb-v2022.10.18/index.md:202]: also mariadb autoscaler

**Review by ArnobKumarSaha — CHANGES_REQUESTED** [review]

File: content/post/kubedb-v2022.10.18/index.md (hunk 1 above)
Comment (spectro30) [reply content/post/kubedb-v2022.10.18/index.md:34]: sounds better. Updated

File: content/post/kubedb-v2022.10.18/index.md (hunk 2 above)
Comment (spectro30) [reply content/post/kubedb-v2022.10.18/index.md:27]: addede

File: content/post/kubedb-v2022.10.18/index.md (hunk 3 above)
Comment (spectro30) [reply content/post/kubedb-v2022.10.18/index.md:181]: Good one. Updated

**Review by ArnobKumarSaha — DISMISSED** [review]

**Review by Shaad7 — APPROVED** [review]
LGTM!

**Pull request merged**